gui/widgets/edittext.cpp

```diff
@@ -74,6 +78,10 @@ void EditTextWidget::handleMouseDown(int x, int y, int button, int clickCount) {
 
 	x += _editScrollOffset;
 
+	if (g_gui.useRTL()) {
+		x = g_gui.getStringWidth(_editString) - x + 14;
```

Comment:
Why `+ 14` ?

Reply:
Added comment explaining why. Quick overview - the click needs to match the exact position where the text starts drawing (when right aligned)